### PR TITLE
Translation Portuguese from Portugal (partial)

### PR DESCRIPTION
**Pull request by qlpqlp**
- Menu translation completed
- Main page Translation completed

### CODE REVIEW

File: i18n/pt-pt.toml [hunk 1]
```diff
@@ -41,162 +41,162 @@ other = "Última actualização"
...
-other = "Visit"
+other = "Visitar"
 
 ["resources[title1]"]
-other = "Foundation"
+other = "Fundação"
 
 ["resources[text1]"]
-other = "Learn more about the Dogecoin Foundation and its Trailmap for Dogecoin!"
+other = "Sabe mais sobre a Fundação Dogecoin e seu Trailmap para Dogecoin."
```
Comment (TheWaveFlows) [inline i18n/pt-pt.toml:173]: Sabe mais sobre a Fundação Dogecoin e o seu Trailmap para Dogecoin.

File: i18n/pt-pt.toml [hunk 2]
```diff
@@ -41,162 +41,162 @@ other = "Última actualização"
...
-other = "Announcements from the Dogecoin Foundation can be found here."
+other = "Anúncios da Fundação Dogecoin podem ser encontrados aqui."
 
 ["resources[title4]"]
-other = "IP FAQs"
+other = "PI FAQs"
 
 ["resources[text4]"]
-other = "Read our responses to some frequently asked questions about Dogecoin IP."
+other = "Leia nossas respostas a algumas perguntas frequentes sobre PI Dogecoin."
```
Comment (TheWaveFlows) [inline i18n/pt-pt.toml:191]: Leia as nossas respostas a algumas perguntas frequentes sobre PI Dogecoin.

File: i18n/pt-pt.toml [hunk 3]
```diff
@@ -206,51 +206,51 @@ other = "Read more"
...
-other = "The Dogecoin Foundation & Dogecoin Project. All rights reserved."
+other = "A Fundação Dogecoin e o Projeto Dogecoin. Todos os direitos reservados."
 
 ##############################
 # wallets/list.html
 ##############################
 
 ["listWallets[muchgoback]"]
-other = "Much Go Back"
+other = "Muito Voltar Atás"
```
Comment (TheWaveFlows) [inline i18n/pt-pt.toml:241]: Muito Voltar Atrás

File: i18n/pt-pt.toml [hunk 4]
```diff
@@ -260,13 +260,13 @@ other = "Download"
 ##############################
 
 ["listContributors[muchgoback]"]
-other = "Much Go Back"
+other = "Muito Voltar Atás"
```
Comment (TheWaveFlows) [inline i18n/pt-pt.toml:263]: Muito Voltar Atrás

**Review by TheWaveFlows — APPROVED** [review]

File: config.toml [hunk 5]
```diff
@@ -597,26 +597,26 @@ disableHugoGeneratorInject = false
 
 [languages.pt-pt.menu]
   [[languages.pt-pt.menu.main]]
-    identifier = "home"
-    name       = "Home"
```
[the remark this answers is not in the document]
Comment (qlpqlp) [reply config.toml:601]: The translation "wow" from Dogecoin silly talk is "uau" thats why I did not add "inicio" :)

File: i18n/pt-pt.toml [hunk 6]
```diff
@@ -41,162 +41,162 @@ other = "Última actualização"
...
-other = "A wallet is necessary for people wanting to use, trade, or hold Dogecoin. You can pick a wallet"
+other = "A carteira é necessária para as pessoas poderem utilizar, trocar ou guardar Dogecoin. Poderás escolher uma carteira"
 
 ["whatisdogecoin[choosewallet][title2]"]
-other = "configure your wallet"
+other = "configurar a tua carteira"
 
 ["whatisdogecoin[choosewallet][text2]"]
-other = "After downloading, you can configure your wallet according to our"
+other = "Após escolheres, poderás configurar a tua carteira de acordo com o nosso"
```
[the remark this answers is not in the document]
Comment (qlpqlp) [reply i18n/pt-pt.toml:103]: The  problem with Download, is, there are Browser wallets that we dont need to download anything :)

**Review by qlpqlp — COMMENTED** [review]
I Applied alsmot all your fixes here https://github.com/dogecoinfoundation/dogecoin.com/pull/151/commits/cfa05d71df85ea6314b4ed9a02151657b6cf5065 :)

**Review by TheWaveFlows — APPROVED** [review]

**Review by TheWaveFlows — APPROVED** [review]

**Review by TheWaveFlows — APPROVED** [review]

**Review by TheWaveFlows — APPROVED** [review]

**Review by TheWaveFlows — APPROVED** [review]

**Review by TheWaveFlows — APPROVED** [review]

**Review by RichieJohn — APPROVED** [review]

**Pull request merged**